sql/core/src/test/scala/org/apache/spark/sql/PlanStabilitySuite.scala

```diff
@@ -207,7 +209,7 @@ trait PlanStabilitySuite extends TPCDSBase with DisableAdaptiveExecutionSuite {
      *   WholeStageCodegen
      *     Project [c_customer_id]
      */
-    def getSimplifiedPlan(node: SparkPlan, depth: Int): String = {
+    def simplifyNode(node: SparkPlan, depth: Int): String = {
```

Comment:
Address the comment from https://github.com/apache/spark/pull/29270#discussion_r476176111